ssl/t1_lib.c

```diff
@@ -3250,9 +3243,7 @@ int tls_choose_sigalg(SSL *s, int fatalerrs)
                         if (!rsa_pss_check_min_key_size(s->ctx, pkey, lu))
                             continue;
                     }
-#ifndef OPENSSL_NO_EC
                     if (curve == -1 || lu->curve == curve)
-#endif
```

Comment:
@mattcaswell, could you double check that this makes sense?  I'm a bit apprehensive about that `curve == -1` check...

Reply:
This all looks correct to me.

Reply:
Okie.  Thanks.